In depth

ddgenome opened the pull request:
@jrday I'd like to merge this in to master and get this published.

ddgenome commented:
I'd also like to rebase to clean up the history on this a bit before merging.

ddgenome commented on docs/user-guide/rug/generators.md:
```diff
@@ -1,104 +1,181 @@
 
-To create a project from this Rug generator, you could run the following
-[rug][cli] command:
+### Using the CLI
```
@Lawouach the characters between the hash marks and the capital "U" is ASCII 160, a "non-breaking space". What editor are you using that is inserting these characters? Any way to turn it off?

ddgenome commented on docs/user-guide/rug/generators.md:
````diff
@@ -0,0 +1,453 @@
+@atomist java spring
+```
+
+### Using the CLI
````
@Lawouach the character between the hash marks and the capital "U" is ASCII 160, a "non-breaking space". What editor are you using that is inserting these characters? Any way to turn it off?

Lawouach replied:
It's more likely the case that I hit the compose key followed by space. My editor should warn me about them indeed.

ddgenome commented:
I've cleaned up the history and worked in @jrday's suggestions.

@Lawouach I moved your commit with the editor content to another branch, rebased on the head of this. Can you work on the branch to reorganize the text of the editor user guide to match organization of the generator user guide?
